apps/ledger-live-desktop/src/config/urls.ts

```diff
@@ -184,6 +184,10 @@ export const urls = {
   howToUpdateNewLedger: "https://support.ledger.com/article/9305992683165-zd",
   genuineCheck: "https://support.ledger.com/article/4404389367057-zd",
   whatAreOrdinals: "https://www.ledger.com/academy/bitcoin-ordinals",
+  ledgerShop:
```

Comment:
IMO this should not be part of this PR (but it's not critical to remove it either)